node/api/channels_api_usage.md

```diff
@@ -1431,7 +1431,7 @@ Open the channel in the same way as in the
 [Initiator WebSocket open](#initiator-websocket-open) example,
 adding the parameters `existing_channel_id` and `offchain_tx` with values
 matching the ones provided in the `leave` report above, additionally add the parameter `existing_fsm_id`
-containing the `fsm_id` received in the [FSM Up](#fsm-up) event. Some parameters (related to open transaction) 
+containing the `fsm_id` received in the [FSM Up](#responder-fsm-is-up) event. Some parameters (related to open transaction) 
```

Comment:
```suggestion
containing the `fsm_id` received in the [FSM Up](#initiator-fsm-is-up) event. Some parameters (related to open transaction)
```

Since this section talks about initiator's parameters, probably the link shall point to the initiator's message. Just a suggestion, not obligatory.